pkg/agent/config.go

```diff
@@ -401,7 +406,8 @@ func ValidateAndCombineConfig(log logr.Logger, cfg Config, flags AgentCmdFlags)
 				" - Use --api-token if you want to use the " + string(JetstackSecureAPIToken) + " mode.\n")
 		}
 		res.AuthMode = mode
-		log.Info(reason)
+		keysAndValues = append(keysAndValues, "mode", mode, "reason", reason)
+		log.V(logs.Debug).Info("Authentication mode", keysAndValues...)
```

Comment:
I changed this to a Debug log, so it will not be shown by default. Why? Because I thought it might be desirable to keep the logs as quiet as possible by default, while allowing these details to be logged in the case of problems.

I'm also not sure if the client-id is sensitive information...is there any risk in logging it?

Reply:
I don't think there is any risk printing out the cliend ID. The client ID is just like a username, the difference being that it isn't a personal information. So I'd say that there is no problem displaying the client ID.